src/components/molecules/NavbarRadio/NavbarRadio.tsx

```diff
@@ -104,13 +104,11 @@ export const NavbarRadio: React.FC = () => {
             })}
             onClick={toggleShowRadioPopover}
           />
-
           <div
-            className="radio-trigger__iframe"
-            style={{
-              visibility: showRadioPopover ? "visible" : "hidden",
-              userSelect: showRadioPopover ? "auto" : "none",
-            }}
+            className={classNames("radio-trigger__iframe", {
+              "radio-trigger__iframe--visible": showRadioPopover,
+              "radio-trigger__iframe--hidden": !showRadioPopover,
+            })}
```

Comment:
Generally preferable to extract the `classNames` function/etc into a `const` in the 'main body' of the component, then use that `const` in the JSX. Makes things cleaner/easier to read.

We usually end up naming this `fooClasses` (`containerClasses` when it's on the main container, some other appropriate name when it's on one of the `__elements`)